### ISSUE TITLE
Every tool group I assign columns to shows up on the annotation page as a tab

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#138]

**Comment by surchs** [comment]
Hey @jarmoza, what would be an ungrouped assessment tool to you? If you mean a tool that's described by only one column, then yes, this should be possible. But I think the way to do that would be to create a tool group that has only one column. There shouldn't be a way to describe an assessment tool column without assigning it to a tool group. Is that what you were asking about?

**Comment by jarmoza** [comment]
> There shouldn't be a way to describe an assessment tool column without assigning it to a tool group. Is that what you were asking about?

Yes, in other words, "Must all assessment tools be grouped?" Can the user decides that they don't want to utilize that functionality?

**Comment by surchs** [comment]
I would say no. As few exceptions as we can get away with. Having a single column tool seems pretty OK to me. Are you thinking of something that would affect the implementation or are you thinking about user experience?

**Comment by jarmoza** [comment]
Okay. So the result of discussion over Slack video is that in order to move onto the Annotation page, all columns that have been categorized as an assessment tool must be part of a group – even if they are a single column.
